### PR TITLE
feat: jx v3 maturity model matrix

### LINKED ISSUE #3069: Jenkins X v3 maturity model
**This is a Feature Request**

**What would you like to be added**
Need to create a Jenkins X v3 maturity model considering relevant evaluative parameters for different cloud providers mainly GKE, AWS and Azure cloud. In addition, evaluate with some of the Version control systems. 

**Why is this needed**
This maturity model / matrix would give the users information about what all features works for the Jenkins X v3 on specific cloud providers, which would help the users get insights on choosing / installing the Jenkins X v3 on cloud.

### PR DESCRIPTION
**Pull request by borntorock**
Resolves #3069 

Signed-off-by: borntorock <borntorock@users.noreply.github.com>

### CODE REVIEW

**Comment by MarckK** [comment]
Awesome work, @borntorock! 

Logs:
Screenshot 2020-10-03 at 14 56 10

**Comment by borntorock** [comment]
> Logs:
> Screenshot 2020-10-03 at 14 56 10

Thanks @MarckK for the logs. Can you point me on how to access the logs of the PR? or does it require some admin privileges that I might not have access. 

Looks like some checks are still not passed after correcting the image location and unable to see the changes deployed in preview environment.

File: content/en/docs/v3/about/maturity-matrix.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Maturity Level Matrix
+linktitle: Maturity Level Matrix 
+description: Jenkins X 3.x features maturity assessment on different cloud providers
+weight: 170
+---
+
+<img src="/static/images/v3/jx-v3alpha-maturity-matrix.png">
```
Comment (ankitm123) [inline content/en/docs/v3/about/maturity-matrix.md:8]: Remove `/static`

**Comment by borntorock** [comment]
Let me know if the maturity matrix here http://jx-docs.jx-jenkins-x-jx-docs-pr-3079.jenkins-x.live/docs/v3/about/maturity-matrix/ looks good or need to adjust the width and height of the image.

**Comment by rawlingsj** [comment]
It's looking great! 

I think we need a comment about still being in alpha but I can help with that first thing Monday. Once that's there I think we are good to merge.

**Pull request merged**